requirements.txt

```diff
@@ -640,6 +640,10 @@ urllib3==2.2.2 \
     #   requests
     #   sentry-sdk
     #   twine
+urlwait==1.0 \
```

Comment:
If we have waitfor, we can build into it everything we're using urlwait for. urlwait is pmac's, but I don't think it's maintained anymore.